packages/sdk-auth/src/auth.js

```diff
@@ -216,21 +230,26 @@ export default class SdkAuth {
 
   anonymousFlow(anonymousId: string = '', config: CustomAuthOptions = {}) {
     const _config = this._getRequestConfig(config)
-    const request = SdkAuth._buildRequest(
+    let request = SdkAuth._buildRequest(
```

Comment:
We build the request, then append to its body and get a new request. That's what I mean by composing.